<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>781</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>741</width>
          <height>61</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QFormLayout" name="createSessionForm">
           <item row="0" column="0">
            <widget class="QLabel" name="filePathLabel">
             <property name="text">
              <string>File Path</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="filePathLineEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="checkpointDirectoryPathLabel">
             <property name="text">
              <string>Checkpoint Directory Path</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="checkpointDirectoryPathLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="Btn_OETest">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Create Session</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTextBrowser" name="Txt_VPBinDataOutput">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>91</y>
          <width>741</width>
          <height>331</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="statusText">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>520</y>
      <width>731</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
   </widget>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOE_Test">
   <property name="text">
    <string>OE Test</string>
   </property>
   <property name="toolTip">
    <string>Test an Omega Edit Feature</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Btn_OETest</sender>
   <signal>clicked()</signal>
   <receiver>actionOE_Test</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
